source/ingress.go

```diff
@@ -109,33 +110,33 @@ func (sc *ingressSource) Endpoints() ([]*endpoint.Endpoint, error) {
 		endpoints = append(endpoints, ingEndpoints...)
 	}
 
+	for _, ep := range endpoints {
+		sort.Sort(ep.Targets)
+	}
```

Comment:
Why this needs to be sorted?
Does anyone rely on having this slice sorted?

Reply:
Currently probably not, but for the planning component it would make sense to have an ordered list of targets. It was just my guess that it is better to have every source return a sorted list oft targets as a convention early on. And on the other it probably won't hurt to have the targets sorted.

Reply:
Ok, I have no real opinion on that.